xdc-plugin/tests/port_indexes.tcl

```diff
@@ -0,0 +1,25 @@
+yosys -import
+plugin -i xdc
+#Import the commands from the plugins to the tcl interpreter
+yosys -import
+
+# -flatten is used to ensure that the output eblif has only one module.
+# Some of symbiflow expects eblifs with only one module.
+synth_xilinx -vpr -flatten -abc9 -nosrl -noclkbuf -nodsp
+
+if {[info procs unknown] != ""} {
+	rename unknown ""
+}
+proc unknown args {puts "'unknown' proc command handler"}
+if {[catch {invalid command}]} {
+	error "Command should be handled by the 'unknown' proc"
```

Comment:
You need to validate that "'unknown' proc command handler"" gets output twice?

Reply:
Done